test/test_helper.rb

```diff
@@ -1,5 +1,5 @@
 require 'test/unit'
-require 'mocha/setup'
+require 'mocha/test_unit'
```

Comment:
required to run tests locally (the mocha version is not specified, and the API has changed)